### PR TITLE
Add admin > mailer page

### PR DESCRIPTION
**Pull request by j0k3r**
Also lock node version to allow gitbook to work

You can see the new page here: https://deploy-preview-178--wallabag-doc.netlify.app/en/admin/mailer

See:
- https://github.com/wallabag/wallabag/issues/6898
- https://github.com/wallabag/wallabag/pull/6899

### CODE REVIEW

File: en/admin/mailer.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- or `sendmail` (using a DSN like `sendmail://default`).
+
+Since 2.6.6, the `gmail` transport is available again using the DSN: `gmail+smtp://USERNAME:PASSWORD@default`
+
+| Name | Description | Default |
+| -----|-------------|-------- |
+| mailer_dsn | One liner with all the mailer parameters `smtp://user:pass@host:465`. Any characters considered special need to be urlencoded in `user`, `pass` and `host`. | smtp://127.0.0.1 |
+
+{% hint style="tip" %}
+Symfony can support other transports which aren't shipped by default to wallabag: Amazon SES, MailChimp, Mailgun, Postmark & SendGrid.
```
Comment (yguedidi) [inline en/admin/mailer.md:14]:
```suggestion
Symfony can support other transports which aren't shipped by default with wallabag: Amazon SES, MailChimp, Mailgun, Postmark & SendGrid.
```

File: en/admin/mailer.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+You can install them using Composer. It's a more complex step to do, [check the Symfony documentation about that](https://symfony.com/doc/4.4/mailer.html).
+{% endhint %}
+
+## Before wallabag 2.6.1
+
+On wallabag < 2.6.1, the mailer was different (we were using an old version of Symfony).
```
Comment (yguedidi) [inline en/admin/mailer.md:21]:
```suggestion
On wallabag < 2.6.1, the mailer was different (we were using an older version of Symfony).
```

**Review by yguedidi — APPROVED** [review]
some suggestions, approved anyway

**Review by nicosomb — APPROVED** [review]

**Comment by nicosomb** [comment]
👏

**Pull request merged**